### PR TITLE
Feat: added accounts page

### PR DESCRIPTION
**Pull request by vinc3nati**
- added profile, address, orders page

image

image

image

### CODE REVIEW

**Review by shrey27 — APPROVED** [review]
code is clean and well structured
UI also looks good
PR is ready to merge

File: src/pages/Profile/ProfileCard.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+export const ProfileCard = ({ title }) => {
+  useDocumentTitle(title);
+  const {
+    user: { user },
+    handleLogOut,
+  } = useAuth();
+  return (
+    <div className="profile-card">
+      <div className="user-img-container">
+        <img className="img img-round" src={UserImage} alt="user-profile" />
```
Comment (OmKakatkar) [inline src/pages/Profile/ProfileCard.jsx:16]: Like the use of alt attribute for images

**Review by OmKakatkar — APPROVED** [review]
Code looks clean
Like the UI

**Review by PeeyushPrashant — APPROVED** [review]
ui looks great
good naming convention

**Pull request merged**